bundles/sirix-core/src/main/java/org/sirix/access/trx/node/AbstractResourceManager.java

```diff
@@ -519,11 +514,10 @@ public void setNodePageWriteTransaction(final @Nonnegative long transactionID, @
    * @throws SirixIOException if an I/O error occurs
    */
   @Override
-  public void closeNodePageWriteTransaction(final @Nonnegative long transactionID) {
+  public void closeNodePageWriteTransaction(final @Nonnegative Long transactionID) {
```

Comment:
Why is it better to use a boxed version over here?

Reply:
@gvdutra Note that if the issue is the auto-boxing happening in `nodePageTrxMap.remove(transactionID)`, I would rather turn `nodePageTrxMap` into a [Long2ObjectMap](https://fastutil.di.unimi.it/docs/it/unimi/dsi/fastutil/longs/Long2ObjectMap.html) as native types are more performant (e.g., there's no extra pointers/space in the heap memory) and we prevent `null` out of the box.

Reply:
As nodePageTrxMap is ConcurrentMap, I was afraid to change it to Long2ObjectMap, I couldn't find the artifact imported, also as I don't know much about the project, I am not sure if Long2ObjectMap would impact in somewhere else, or even if it is threadsafe!

Reply:
I'm also not sure, but it has to be threadsafe. Maybe just leave it as of now?